### PR TITLE
Py-test Pr review additions

### PR DESCRIPTION
**Pull request by stymoshchuk**

### CODE REVIEW

File: unit/models/returnAch.py [hunk 1]
```diff
@@ -5,10 +5,9 @@
     from typing_extensions import Literal
 from unit.models import *
 
-AchReturnReason = Literal["Unauthorized"]
+AchReturnReason = Literal["InsufficientFunds", "Unauthorized", "UncollectedFunds"]
 
-
-class ReturnReceivedAchTransactionRequest(object):
+class ReturnReceivedAchTransactionRequest(UnitRequest):
```
Comment (axshani) [inline unit/models/returnAch.py:10]: good inheritance, now we can use to_payload()

File: unit/models/statement.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def from_json_api(_id, _type, attributes, relationships):
 
 OutputType = Literal["html", "pdf"]
 
-class GetStatementParams(object):
+class GetStatementParams(UnitRequest):
```
Comment (axshani) [inline unit/models/statement.py:19]: you are not wrong, but in this case UnitRequest doesn't help too much. your decision

**Review by axshani — CHANGES_REQUESTED** [review]

File: unit/models/payment.py [hunk 3]
```diff
@@ -41,11 +42,13 @@ def __init__(self, interval: str, day_of_month: int, start_time: Optional[dateti
 class BasePayment(object):
     def __init__(self, _id: str, created_at: datetime, status: PaymentStatus, direction: PaymentDirections, description: str,
                  amount: int, reason: Optional[str], tags: Optional[Dict[str, str]],
-                 relationships: Optional[Dict[str, Relationship]]):
+                 relationships: Optional[Dict[str, Relationship]], astra_routine_id: Optional[str] = None):
         self.id = _id
         self.attributes = {"createdAt": created_at, "status": status, "direction": direction,
                            "description": description, "amount": amount, "reason": reason, "tags": tags}
         self.relationships = relationships
+        if astra_routine_id:
```
[the remark this answers is not in the document]
Comment (stymoshchuk) [reply unit/models/payment.py:50]: i will removee

File: unit/models/payment.py [hunk 4]
```diff
@@ -419,9 +434,45 @@ def to_json_api(self) -> Dict:
...
+            self.amount = amount
+            self.send_date = send_date
+            self.counterparty = counterparty
+            self.memo = memo
+            self.idempotency_key = idempotency_key
+            self.tags = tags
+            self.relationships = relationships
+
+        def to_json_api(self) -> Dict:
+            payload = super().to_payload("checkPayment", self.relationships)
```
[the remark this answers is not in the document]
Comment (stymoshchuk) [reply unit/models/payment.py:466]: remove self.rel

File: unit/models/payment.py [hunk 5]
```diff
@@ -419,9 +434,45 @@ def to_json_api(self) -> Dict:
         payload["data"]["attributes"]["counterparty"] = self.counterparty
         return payload
 
+class CreatePushToCardPaymentRequest(CreatePaymentBaseRequest):
+        def __init__(self, amount: int, description: str, configuration: dict,
+                     relationships: Dict[str, Relationship],
+                     idempotency_key: Optional[str] = None, tags: Optional[Dict[str, str]] = None):
+            super().__init__(amount, description, relationships, idempotency_key, tags, None, "pushToCardPayment",
```
[the remark this answers is not in the document]
Comment (stymoshchuk) [reply unit/models/payment.py:441]: finish it

**Pull request merged**